Source/CsDebugScript.Engine/Engine/Debuggers/DbgEngDll.cs

```diff
@@ -1243,6 +1260,81 @@ public DebugEventInfo GetLastEventInfo()
             }
         }
 
+        /// <summary>
+        /// Releases the process that is being debugged.
+        /// </summary>
+        public void ContinueExecution(Process process)
+        {
+            using (var processSwitcher = new ProcessSwitcher(StateCache, process))
+            {
+                DebuggeeFlowController flowControler = debugeeFlowControlers[process.Id];
+                flowControler.DebugStatusBreak.WaitOne();
+                Control.Execute(0, "g", 0);
+            }
+        }
+
+        /// <summary>
+        /// Continues execution of current process.
+        /// </summary>
+        public void ContinueExecution()
+        {
+            ContinueExecution(Process.Current);
+        }
+
+        /// <summary>
+        /// Breaks the process that is being debugged.
+        /// </summary>
+        public void BreakExecution(Process process)
+        {
+            using (var processSwitcher = new ProcessSwitcher(StateCache, process))
+            {
+                DebuggeeFlowController flowControler = debugeeFlowControlers[process.Id];
+                flowControler.DebugStatusBreak.Reset();
+                Control.SetInterrupt(0);
+                flowControler.DebugStatusBreak.WaitOne();
+
+                // Drop the cache.
+                //
+                process.InvalidateProcessCache();
```

Comment:
Please add TODO: for this line

TODO: If break actually breaks all processes, we need to clear cache for all of them.